Makefile

```diff
@@ -19,7 +27,10 @@ test-layer1:
 	make -C layer1 test
 	make -C integration_tests test
 
-test: test-layer1 test-layer2
+tezt: build
+	dune exec tezt/tezt_chusai.exe -- --verbose
+
+test: test-layer1 test-layer2 tezt
```

Comment:
`make tezt` didn't seem to work on my local

```
[09:07:58.390] [error] node1 exited with code 1 (full command: ./_build/tezos-node identity generate --data-dir /tmp/tezt-465434/1/node1 0)
[09:07:58.390] [FAILURE] (1/1, 1 failed) Jakarta: A dummy test that expect every smart-contracts can be originate and that the state is properly initiated
```

Reply:
I made some clarification about how to get tezos binaries on the readme, if you follow the instruction does it work?

Reply:
Errors:
```
[06:24:23.183] Initialize Node and Client
[06:24:23.184] [node1] ./_build/tezos-node identity generate --data-dir /tmp/build_920df3_dune/tezt-589647/1/node1 0
[06:24:23.186] node1 exited with code 127.
[06:24:23.186] [error] node1 exited with code 127 (full command: ./_build/tezos-node identity generate --data-dir /tmp/build_920df3_dune/tezt-589647/1/node1 0)
[06:24:23.186] [FAILURE] (1/1, 1 failed) Jakarta: A dummy test that expect every smart-contracts can be originate and that the state is properly initiated
[06:24:23.186] Try again with: ./tezt_chusai.exe --verbose --file dummy_test.ml --title 'Jakarta: A dummy test that expect every smart-contracts can be originate and that the state is properly initiated'
File "tezt/dune", line 2, characters 7-18:
2 |  (name tezt_chusai)
           ^^^^^^^^^^^
Command exited with code 1.
```

There two error there:
1. `dune runtest --no-buffer -j 1` doesn't seem to be able to run `tezos-node identity generate  ...`.
```
[06:24:23.184] [node1] ./_build/tezos-node identity generate --data-dir /tmp/build_920df3_dune/tezt-589647/1/node1 0
``` 
(a) If I run the command `tezos-node identity generate  ...` manually, it tells me I need to install Zcash stuff (see [here](https://tezos.stackexchange.com/questions/3185/cannot-find-zcash-params)). 
(b) After installing zcash stuff, I still get the same error.
(c) But if I run the `tezos-node identity generate  ...` manually again, it gens identity and `make test-layer2` works.

2. If the above is fixed, this one still remains.
```
File "tezt/dune", line 2, characters 7-18:
2 |  (name tezt_chusai)
           ^^^^^^^^^^^
Command exited with code 1.
```

Reply:
It is weird. @Hakimba seems suffer of the same issue. 
Does @pecornilleau find a way to fix the suite?

Reply:
[feac9d7](https://github.com/marigold-dev/chusai/pull/43/commits/feac9d7cb76c07acaedd0c9ae48bd082a2012c06) fix that.

Reply:
^ can confirm, works for me

Reply:
works for me, too.